.github/workflows/build.yml

```diff
@@ -2,13 +2,13 @@
 name: build
 
 on:
-  push
+  pull_request: ~
+  push:
+    branches: [master]
 
 jobs:
   build:
     runs-on: ubuntu-20.04
-    env:
-      GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think we still need this?

Reply:
I'm honestly not sure what you'll need this for here...

Reply:
Ah okay didn't realize something else built the image.